mastering-plone/configuring_customizing.rst

```diff
@@ -82,6 +82,8 @@ Let's change the logo.
 * Go to http://localhost:8080/Plone/@@site-controlpanel
 * Upload the Logo.
 
+.. image:: _static/configuring_customizing_logo.png
```

Comment:
It would be great to use a better logo that the old one from 2014. One that we can keep for a couple of years 😄

Reply:
Is it possible to change the size of the logo? I changed it but I think it's to big...